### DISCUSSION TITLE
How can i find source code for a spesific version?

### DISCUSSION THREAD

**Discussion by alexen80** [shinyorg/shiny#1541] [Q&A]
Sorry if this is a stupid question. I just dont understand how i can find the source for version 3.3.3 "NuGet\Install-Package Shiny.Core -Version 3.3.3" or other versions. The latest i can find in releases seem to be 3.1, and 2.7 missing. I can find 2.7 in branches, but not 3.3. Am I missing something obvious here? Access to source is really valuable for debugging.

**Comment by aritchie** [comment]
v3.x are all merged into the v3.0 branch
